packages/knip/test/plugins/dependency-cruiser.test.ts

```diff
@@ -0,0 +1,27 @@
+import { test } from 'bun:test';
+import assert from 'node:assert/strict';
+import { main } from '../../src/index.js';
+import { resolve } from '../../src/util/path.js';
+import baseArguments from '../helpers/baseArguments.js';
+import baseCounters from '../helpers/baseCounters.js';
+
+const cwd = resolve('fixtures/plugins/dependency-cruiser');
+
+test('Find dependencies with the dependency-cruiser plugin', async () => {
+  const { issues, counters } = await main({
+    ...baseArguments,
+    cwd,
+  });
+
+  // Check that dependencies are properly detected
+  assert(issues.devDependencies['package.json']['dependency-cruiser']);
+  assert(issues.binaries['package.json']['depcruise']);
```

Comment:
what i usually do is add the most minimal package in an actual `node_modules` folder with e.g. `{ "name": "dependency-cruiser", "bin": { ... } }` in `package.json` and an empty `index.js` - that way, Knip does the full machinery and there should be no issues left (random example: https://github.com/webpro-nl/knip/tree/main/packages/knip/fixtures/plugins/angular/node_modules/@angular/cli).

Reply:
That's awesome. I was a bit lost working on the test